src/problem-analysis/state-of-the-art/SOTA.bib

```diff
@@ -6,3 +6,48 @@ @report{alpha-go-zero
     year = {2018},
     pages = {32},
 }
+
+@misc{chess-com-chess-engines,
+    author = {chess.com and others},
+    title = {Computer Chess Engines: A Quick Guide},
+    url = {https://www.chess.com/article/view/computer-chess-engines},
+}
+
+@misc{atoms,
+    author = {John Carl Villanueva},
+    title = {Atoms in the universe},
+    url = {https://www.universetoday.com/36302/atoms-in-the-universe/},
+}
+
+
+@misc{zero-sum-games,
+    author = {Janet Chen, Su-I Lu, Dan Vekhter},
```

Comment:
@SebastianMygind Use `and` instead of `,`. Otherwise the entry looks weird:
<img width="814" alt="image" src="https://github.com/audio-engineer/aau-p2-article/assets/70742985/1c8d2619-2438-418c-82de-3cbd2bc9ac5f">